build.py

```diff
@@ -0,0 +1,37 @@
+import shutil
+import sass
+import glob
+
+from pathlib import Path
+from staticjinja import Site
+from yaml import load, Loader
+
+def data():
+    data_dict = {}
+    yaml = glob.glob("data/*.yaml")
```

Comment:
pathlib objects have built in glob. You already imported it so maybe use it too?